### PR TITLE
modified .css elements of pages

### PR DESCRIPTION
**Pull request by GuaBaiChi**
hoi hoi hoi

### CODE REVIEW

File: frontend/src/Components/Content/Brigand.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+export default function Brigand() {
+  return (
+      <>
+          <div style={AlbumNameAlignStyle}>
+              <div style={AlbumNameStyle} translate="no">
+                  Brigand Album Name
```
Comment (H0r53) [inline frontend/src/Components/Content/Brigand.tsx:6]: TheVault

File: frontend/src/Components/Content/Brigand.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+              </div>
+              <div style={AlbumNameSpacingStyle} translate="no">
+              </div>
+          </div>
+          <div>
+              <img className="artCover hyperShadow pointer" style={AlbumArtStyle} src={process.env.PUBLIC_URL + "/img/TafAlbumCover.png"} />
+          </div>
```
[the remark this answers is not in the document]
Comment (GuaBaiChi) [reply frontend/src/Components/Content/Brigand.tsx:13]: BUSTED for lazy copy/paste hack job.

Okay I can't find  `/img/TafAlbumCover.png` in any of the folders.

File: frontend/src/Components/Content/Brigand.tsx (hunk 1 above)
Comment (GuaBaiChi) [reply frontend/src/Components/Content/Brigand.tsx:6]: Changed.

File: frontend/src/Components/Content/Brigand.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (GuaBaiChi) [reply frontend/src/Components/Content/Brigand.tsx:13]: resolved by copy/pasting the_vault.jpg from /backend to /frontend

`"/img/the_vault.jpg"`

File: frontend/src/Components/Content/Brigand.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      <>
+          <div style={AlbumNameAlignStyle}>
+              <div style={AlbumNameStyle} translate="no">
+                  Brigand Album Name
+              </div>
+              <div style={AlbumNameSpacingStyle} translate="no">
+              </div>
+          </div>
+          <div>
+              <img className="artCover hyperShadow pointer" style={AlbumArtStyle} src={process.env.PUBLIC_URL + "/img/TafAlbumCover.png"} />
```
Comment (jacobmillsfl) [inline frontend/src/Components/Content/Brigand.tsx:12]: This appears to be using a TAF album cover image

**Pull request merged**